### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2291

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/migration-interop-guidance-for-teams-with-skype.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ When a user is in any of the Skype for Business modes (SfBOnly, SfBWithTeamsColl
...
...ms by another user in their Teams client if they are in the *same tenant*.<li>Receives chats & VoIP calls initiated in Teams by another user in their Skype for Business client if they are in a  *federated tenant*. <li>Has PSTN functionality as noted below:<ul><li>When the user is homed in Skype for Business on-premises and has Enterprise Voice, PSTN calls are always initiated and received in Skype for Business.<li>When the user is homed on Skype for Business Online and has Microsoft Phone System, the user always initiates and receives PSTN calls in Skype for Business:<ul><li>This happens regardless of whether the user has a Microsoft Calling Plan, or connects to the PSTN network via either Skype for Business Cloud Connector Edition or an on-premises deployment of Skype for Business Server (hybrid voice)<li>**Note: Microsoft Teams Phone System Direct Routing is not supported in Islands mode.**</ul></ul><li>Receives Microsoft Call Queues and Auto-Attendant calls in Skype for Business<li>Can schedule meetings in Teams or Skype for Business (and will see both plug-ins by default).<li>Can join any Skype for Business or Teams meeting; the meeting will open in the respective client.</ul>|
```
Comment (JohanFreelancer9) [inline Teams/migration-interop-guidance-for-teams-with-skype.md]:
```suggestion
|**Islands**</br>(default)|A user runs both Skype for Business and Teams side-by-side. This user:</br><ul><li>Can initiate chats and VoIP calls in either Skype for Business or Teams client. Note: Users with Skype for Business homed on-premises cannot initiate from Teams to reach another Skype for Business user, regardless of the recipient's mode.<li>Receives chats & VoIP calls initiated in Skype for Business by another user in their Skype for Business client.<li>Receives chats & VoIP calls initiated in Teams by another user in their Teams client if they are in the *same tenant*.<li>Receives chats & VoIP calls initiated in Teams by another user in their Skype for Business client if they are in a  *federated tenant*. <li>Has PSTN functionality as noted below:<ul><li>When the user is homed in Skype for Business on-premises and has Enterprise Voice, PSTN calls are always initiated and received in Skype for Business.<li>When the user is homed on Skype for Business Online and has Microsoft Phone System, the user always initiates and receives PSTN calls in Skype for Business:<ul><li>This happens regardless of whether the user has a Microsoft Calling Plan, or connects to the PSTN network via either Skype for Business Cloud Connector Edition or an on-premises deployment of Skype for Business Server (hybrid voice).<li>**Note: Microsoft Teams Phone System Direct Routing is not supported in Islands mode.**</ul></ul><li>Receives Microsoft Call Queues and Auto-Attendant calls in Skype for Business.<li>Can schedule meetings in Teams or Skype for Business (and will see both plug-ins by default).<li>Can join any Skype for Business or Teams meeting; the meeting will open in the respective client.</ul>|
```

**Comment by LolaJacobsen** [comment]
@bjwhalen and @CarolynRowe - for your review.

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
@scanum, @nenonix, I've read through the updated copy and the original and don't see the text that was updated. Can you point out the changes to me?  Thanks.

**Comment by CarolynRowe** [comment]
This request needs to be reviewed by @bjwhalen before we merge.

**Comment by scanum** [comment]
@bjwhalen Hi! Could you please review? Thanks!

**Pull request merged**